imports/ui/stylesheets/_variables.scss

```diff
@@ -2,6 +2,7 @@
 $primary-color: #008cba !default; // bondi blue
 $secondary-color: #cf515f !default; // chestnut rose
 $text-color: #333333 !default; // mine shaft
+$text-color-light: #ffffff !default; // white
 
 $background-color: #F4F5F8 !default; // athens grey

```

Comment:
From the Sass documentation:
> You can assign to variables if they aren’t already assigned by adding the !default flag to the end of the value. This means that if the variable has already been assigned to, it won’t be re-assigned, but if it doesn’t have a value yet, it will be given one.

Heißt: Rebranding wird erleichtert. -- Wollen wir das?

Vielleicht, aber `!default` flags sehen hübsch aus. Wenn sie keinen wirklichen Sinn verfolgen, so doch wenigstens einen ästhetischen im Code.